presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/EncoderColumnHandle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.encoder;
+
+import io.prestosql.spi.connector.ColumnHandle;
+import io.prestosql.spi.type.Type;
+
+public interface EncoderColumnHandle
+        extends ColumnHandle
+{
+    boolean isInternal();
```

Comment:
We can drop this.

Reply:
This method is used in `KafkaPageSinkProvider#createPageSink` to check for internal columns

Reply:
Of course, I don't know why I suggested that.